slsim/Sources/point_plus_extended_source.py

```diff
@@ -0,0 +1,83 @@
+import numpy.random as random
+from slsim.Sources.source_pop_base import SourcePopBase
+import numpy as np
+from slsim.selection import deflector_cut
+from slsim.Sources.galaxies import galaxy_projected_eccentricity
+
+
+class PESource(SourcePopBase):
```

Comment:
I would use a more meaningful class name

Reply:
Changed to PointPlusExtendedSource.